app/mailers/club_application_mailer.rb

```diff
@@ -1,5 +1,5 @@
 class ClubApplicationMailer < ActionMailer::Base
-  default from: 'Zach Latta <team@hackedu.us>'
+  default from: 'Zach Latta <team@hackclub.io>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.